.travis.yml

```diff
@@ -14,32 +14,22 @@ services:
 
 env:
   matrix:
-  - NODE_VERSION=8 COUCH_VERSION=2
-  - NODE_VERSION=6.10 COUCH_VERSION=1
+  - NODE_VERSION=8
+  - NODE_VERSION=6.10
   global:
   - COUCH_URL=http://admin:pass@localhost:5984/medic
   - API_URL=http://admin:pass@localhost:5988
   - COUCH_NODE_NAME=nonode@nohost
-  #AWS Access Key Id and AWS Secret Access Key encrypted by travis to access the S3 buckets where reports and screenshots are saved 
+  #AWS Access Key Id and AWS Secret Access Key encrypted by travis to access the S3 buckets where reports and screenshots are saved
   - secure: TR1UN2r3beDtIF+VJpLF2ocTv/uxuOKyVWWhzMLre0ZrrBaIP1sLZV7Z4S/km5M92EfPgGW87BogdWE/R+kTTRiPTCbFB/U/3jFKxEZRXKST66YK5JMwsYqb17UtZtFdqEtO9GGbAVzXwpZfMMoXvlKNrors2W32xBm2uIkOSpI=
   - secure: Q8RH65NClRBryxfvlwHQjeR4wGs+GXeUBRIBd1kspAM7Uv+5K3iP+q/TPCIrTL/OpJ5tyCYfeq5hKIFFwrY3JikNXnyHICOjXPE6zSLUm8E8NHaP/orPNWjze2x4yDSRCifqzr2ZiXVq9sxlZfNbZ9eyJxpPFTYDSpuN9T5UEE0=
 
 before_install:
-  - |
-    if [ "$COUCH_VERSION" = 2 ]; then
-      # start couchdb 2.x docker instance
-      docker run -d -p 5984:5984 --name couch klaemo/couchdb:2.0.0
-      echo "Starting CouchDB 2.0"
-    elif [ "$COUCH_VERSION" = 1 ]; then
-      # because we're on a java project we don't get CouchDB1.x for free
-      sudo apt-get install -y couchdb
-      # though sometimes we may do, so make sure it's running
-      sudo service couchdb start
-      until nc -z localhost 5984; do sleep 1; done
-      echo "Starting CouchDB 1.x started"
-    fi
-    until nc -z localhost 5984; do sleep 1; done
-    echo "CouchDB Started"
+  # start couchdb 2.x docker instance
+  - docker run -d -p 5984:5984 --name couch klaemo/couchdb:2.0.0
```

Comment:
We should update this to the official docker image and also update the version to 2.1.1 which is what we're actually going to deploy against. I think this will work...

```
 - docker run -d -p 5984:5984 --name couch apache/couchdb:2
```

This will keep on the latest 2.x version which will match the support matrix of 2.1.1 or above.

Reply:
Cool I'll give that a shot.